sdgx/data_processors/generators/email.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import pandas as pd
+from faker import Faker
+
+from sdgx.data_models.metadata import Metadata
+from sdgx.data_processors.extension import hookimpl
+from sdgx.data_processors.generators.pii import PIIGenerator
+
+fake = Faker(locale="zh_CN")
```

Comment:
Should users be allowed to configure the `local`? Although `email` has no specific `Providers`

Reply:
The local configuration probably doesn't work for email. I will modify this fake object.